test/suites/optional/generic-device-plugin.robot

```diff
@@ -49,6 +49,29 @@ Verify ttyUSB glob pattern device discovery and allocation
 
     [Teardown]    TtyUSB Glob Test Teardown
 
+Verify FUSE device allocation and accessibility
+    [Documentation]    Verifies FUSE device configuration, allocation, and accessibility in pods
+    [Tags]    fuse-device
+    Enable And Configure GDP    ${GDP_CONFIG_FUSE_COUNT}
+    Wait Until Device Is Allocatable    10    fuse
+    # Create and deploy pod requesting 4 FUSE devices
+    ${path}=    Create Random Temp File    ${POD_FUSE_DEVICE}
+    Oc Create    -f ${path} -n ${NAMESPACE}
+
+    # Wait for pod to be ready and verify device accessibility
+    Oc Wait    -n ${NAMESPACE} pod/fuse-test-pod    --for=condition=Ready --timeout=120s
+
+    # Verify /dev/fuse is accessible in the pod
+    ${fuse_device}=    Run With Kubeconfig    oc exec -n ${NAMESPACE} fuse-test-pod -- ls -l /dev/fuse
+    Should Contain    ${fuse_device}    /dev/fuse
+
+    # Verify node allocation shows 4 FUSE devices allocated
+    ${node}=    Run With Kubeconfig    oc get node -o=name
+    ${node_name}=    Remove String    ${node}    node/
+    ${describe_output}=    Run With Kubeconfig    oc describe node ${node_name}
+    Should Contain    ${describe_output}    device.microshift.io/fuse:    10
+    Should Contain    ${describe_output}    device.microshift.io/fuse    4    4
```

Comment:
There are 2+ spaces after fuse and numbers, this means they are separate arguments to the keyword
```
Should Contain    ${describe_output}    device.microshift.io/fuse    4    4
                  --- arg 1 -------    --- arg 2 -----------       arg3  arg4
```

You can see it in the execution log:
```
Arguments: [ ... | 'device.microshift.io/fuse' | '4' | '4' ]
```

I think it skips the fours and just matches the first string. I think we need to create string with 4 spaces inside in another way